templates/Start/index.html.twig

```diff
@@ -24,6 +24,7 @@
                 <img src="{{  asset('build/images/diff-added.svg') }}" alt="Invite Codes">
                 <div class="caption">
                     <h3>{{ "start.vouchers"|trans }}</h3>
+                    <p>{{ "start.vouchers-desc"|trans }}</p>
```

Comment:
Should be `<p class="alert alert-info">`, no?

Reply:
No, this is the subtitle of the voucher feature on the index page.